src/commands/write.ts

```diff
@@ -23,7 +24,7 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
   }) as yargs.Argv<Options>;
 
 export const handler = async (argv: Arguments<Options>): Promise<void> => {
-  const { statement, privateKey, chain } = argv;
+  const { statement, privateKey, chain, rpcRelay } = argv;
```

Comment:
here we're testing relaying (on by default) so that in theory, folks could test out making their write calls directly using their specified provider.